src/pybind/mgr/rbd_support/mirror_snapshot_schedule.py

```diff
@@ -260,17 +274,17 @@ def handle_get_mirror_mode(self, image_spec, image, comp, mode):
                 "CreateSnapshotRequests.handle_get_mirror_mode: {}/{}/{}: {}".format(
                     pool_id, namespace, image_id,
                     "snapshot mirroring is not enabled"))
-            self.close_image(image_spec)
+            self.close_image(image_spec, image)
```

Comment:
@tchaikov Could you please also add `return` after `close_image`? We should return if we ended up here instead of continuing.  I think this is what  @dillaman meant when was talking about "missing a return statement".

Reply:
@trociny sure. added in a separate commit.